perf-harness/src/test/java/io/opentelemetry/perf/OtlpPipelineStressTest.java

```diff
@@ -101,6 +102,8 @@
 
   private final InMemoryMetricExporter metricExporter = InMemoryMetricExporter.create();
 
+  private SdkTracerProvider tracerProvider;
```

Comment:
Do we need the whole `SdkTracerProvider` here, or is the management interface enough?  I actually struggled with this kind of decision in the examples, as well.  It's so easy to end up with the SDK tracer provider and to use it for instrumentation, and we'd definitely like to discourage that pattern as much as possible.